util/pace/util/logging.py

```diff
@@ -1,15 +1,27 @@
 import logging
+import os
 import sys
 
 from mpi4py import MPI
 
 
+LOGLEVEL = os.environ.get("PACE_LOGLEVEL", "INFO").upper()
+
+AVAILABLE_LOG_LEVELS = {
+    "info": logging.INFO,
+    "debug": logging.DEBUG,
+    "warning": logging.WARNING,
+    "error": logging.ERROR,
+    "critical": logging.CRITICAL,
+}
```

Comment:
I wish my python skills were better so I wouldn't have to ask this question...

Are the log levels designed to be independent or are they hierarchical, meaning setting ERROR level also outputs all CRITICAL messages too?  If independent, I'm assuming I can set multiple levels at once.

Reply:
They are indeed hierarchical so ERROR includes CRITICAL.

Reply:
Can you add that info as a comment in the logging.py file - thx